redhat-developer/app-services-cli
# Connectors start stop

This pr adds two new commads for connectors, start and stop. Use to toggle connector instance on and off.

### Verification Steps
Create any connector instance and make sure it is working and in the ready state.
1. run `rhoas context set-connector` to set that connector to the current connector in the context
2. run `rhoas connector stop` then check in the UI if the state of the connector is stopped
3. run `rhoas connector start` then check in the UI if the state of the connector is started again
4. if you have multiple connectors try use `rhoas connector start/stop --id=...` to toggle the state of any connector without using context

### Type of change
